sdk/storage/storage-file-datalake/test/filesystemclient.spec.ts

```diff
@@ -130,6 +131,25 @@ describe("DataLakeFileSystemClient", () => {
     assert.deepEqual(result.metadata, metadata);
   });
 
+  it("create with encryption scope", async function (this: Context) {
+    let encryptionScopeName;
+    try {
+      encryptionScopeName = getEncryptionScope();
+    } catch {
+      this.skip();
```

Comment:
Why skip?  Shouldn't we let the test fail and fix the test environment instead?

Reply:
I'll make a new PR to update this and the ones in blob package.